katas/content/superposition/greenberger_horne_zeilinger/solution.md

```diff
@@ -0,0 +1,26 @@
+### Solution
+
+The single-qubit GHZ state is the plus state $\frac{1}{\sqrt{2}} \big (|0\rangle + |1\rangle\big)$ that we've discussed in task 1.1. As a reminder, that state is prepared by applying a Hadamard gate.
+
+The 2-qubit GHZ state is the Bell state $\frac{1}{\sqrt{2}} \big (|00\rangle + |11\rangle\big)$ that we've discussed in task 1.6.
```

Comment:
"in the eponymous task" is probably too fancy? :-) Maybe "in the two previous tasks"?

Reply:
hah yes let's go with 'in the two previous tasks'